### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by someonetang** [criyle/go-judge#69] [Q&A] [answered]
我在用apifox做尝试在wsl上运行的接口链接的时候返回如下：
[
    {
        "status": "Internal Error",
        "exitStatus": 0,
        "error": "execve: start: execve: no such file or directory",
        "time": 196200,
        "memory": 131072,
        "runTime": 0,
        "files": {
            "stderr": "",
            "stdout": ""
        },
        "fileIds": {
            "a.cc": "LKOY2DCI"
        },
        "fileError": [
            {
                "name": "a",
                "type": "CopyOutOpen",
                "message": "open a: no such file or directory"
            }
        ]
    }
]，
请问可能是什么原因呢，文档中这一部分没有看明白

**Comment by criyle** [comment, marked as answer]
可能args参数指定的二进制文件路径在沙箱中不存在。比方说没有安装g++时尝试运行该程序。
